Aptoide/aptoide-client-v8
# [AND-7] Update third party libs

**What does this PR do?**

   - Updates all libs for the current version of AG.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [AND-7](https://aptoide.atlassian.net/browse/AND-7)

- Check if the project compiles and runs correctly

**What are the relevant tickets?**

  Tickets related to this pull-request: [AND-7](https://aptoide.atlassian.net/browse/AND-7)

**Code Review Checklist**


[AND-7]: https://aptoide.atlassian.net/browse/AND-7?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AND-7]: https://aptoide.atlassian.net/browse/AND-7?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
